app/components/Layout/Header.tsx

```diff
@@ -1,16 +1,16 @@
 import React, {useState, useRef} from 'react';
-import {Form} from 'react-bootstrap';
 import Link from 'next/link';
 import LoginButton from 'components/LoginButton';
 import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
-import {faBars} from '@fortawesome/free-solid-svg-icons';
+import {faBars, faUser} from '@fortawesome/free-solid-svg-icons';
 
 const DESKTOP_BREAKPOINT_QUERY = '(min-width: 992px)';
 
 const HeaderLayout = ({
   isLoggedIn = false,
   isRegistered = false,
   fixed = false,
+  userProfileDropdown = null,
   children
 }) => {
```

Comment:
Look like this component is missing some typed Props 
```suggestion
interface Props {
  isLoggedIn?: boolean;
  isRegistered?: boolean;
  fixed?: boolean;
  userProfileDropdown?: React.ReactNode;
}
const HeaderLayout: React.FunctionComponent<Props> = ({
  isLoggedIn = false,
  isRegistered = false,
  fixed = false,
  userProfileDropdown,
  children
}) => {
```